My LandingPage

TaniaP26 opened the pull request:
https://TaniaP26.github.io/layout_landing-page

volodymyr-soltys97 reviewed (changes requested):
Good job 👍 
Let's improve your page
1. Add a favicon on the page and change title to `The MET` 
<img width="152" alt="image" src="https://github.com/user-attachments/assets/9c7cfd45-ed00-4d3b-88c6-fe94ae66664b" />

2. Add hover effects for all clickable elements on the page 
<img width="142" alt="image" src="https://github.com/user-attachments/assets/98bc7e46-3911-481e-a3e1-1119eb8b0d89" />

3. You need to show additional block with phone number when hovering this icon and make a call when user clicks here 
<img width="68" alt="image" src="https://github.com/user-attachments/assets/b372c0da-d7e8-4699-9c40-62a81e75e43a" />

4. Add transition when this menu appears on the page and remove scroll when menu is open 
<img width="1512" alt="image" src="https://github.com/user-attachments/assets/037f6ffe-02bd-44bb-9ca3-5b0a60efaf51" />

5. The images shouldn't overlap each other when hovering 
<img width="1509" alt="image" src="https://github.com/user-attachments/assets/5e8bf06e-1ec2-4f14-92d4-ff11bf8c7785" />

6. These elements should be links and add hover effects here 
<img width="741" alt="image" src="https://github.com/user-attachments/assets/5144f8f3-1d95-4fda-bb02-07eebcf5babb" />

TaniaP26 commented:
Thanks for comments. I fixed everything almost but one. I don't know how  remove scroll when menu is open because this scroll belongs to the page__body class. I just added  z-index=100 to the menu.

anastasiiavorobiova reviewed (approved):
Great work!

TaniaP26 commented:
Thanks!
